_schema.graphql

```diff
@@ -5277,6 +5277,31 @@ type DaySchedule {
   day_of_week: String
 }
 
+# Autogenerated input type of DebitCommissionExemption
+input DebitCommissionExemptionInput {
+  # A unique identifier for the client performing the mutation.
+  clientMutationId: String
+
+  # Gross sale exemption amount and sale currency. Amount must be positive
+  exemption: MoneyInput!
+  notes: String
+
+  # Partner ID
+  partnerId: String!
+
+  # The transaction ID (e.g. Exchange order ID) associated with this change
+  referenceId: ID
+}
+
+# Autogenerated return type of DebitCommissionExemption
+type DebitCommissionExemptionPayload {
+  # The amount of GMV in sale currency to not charge commission on in this transaction
+  amountOfExemptGmvOrError: MoneyOrErrorUnion!
+
+  # A unique identifier for the client performing the mutation.
+  clientMutationId: String
+}
+
```

Comment:
There's a _lot_ more in the schema now that I wouldn't necessarily expect to see. Was adding all this intentional or just a consequence of copying over the gravity schema?

Reply:
This is the V1 schema. 😬. Ignore.

Reply:
@zephraph I think these changes are in the v2 schema as well though - this was all work that was done in Gravity back in Jan/Feb that we never added to MP (we decided at the time to hit Gravity's GQL endpoint directly from Exchange), so my guess is that MP's Gravity schema just hasn't been updated since then.